### ISSUE TITLE
Schedule issue

### ISSUE THREAD

**Issue by ZuggieTM** [gergo85/oc-news#103]
What does the schedule exactly do? It's set to a connection named newsletter, but I don't see that connection anywhere? Can you elaborate on that, cause it's causing issues on my server and I was wondering if I could just delete the schedule.

**Comment by TimFoerster** [comment]
The scheduler is an ocms feature: 
https://octobercms.com/docs/setup/installation#crontab-setup

The scheduler executes the `queue:work` command in the `Plugin.php` file which allows queued newsletter sending.

if you ssh into your server and execute `crontab -e` you can remove the `* * * * * php /path/to/artisan schedule:run >> /dev/null 2>&1` line to get rid off the scheduler.

This obv. removes all queued/jobs features of ocms.

But it would be better to fix your issue. What is causing the problem and which error do you get?

**Comment by ZuggieTM** [comment]
Yeah sorry, my explanation was terrible. I know what Cron and scheduler is. What I'm curious about is the connection newsletter. If I wanna use it, I have to set it up in config/queue.php?
I have the issue that my server was running in 14 times. Or well, 12 was hagning because 1 was already running. It seems odd when it has the withoutOverlapping().

**Comment by TimFoerster** [comment]
You simply have to do that: 
https://octobercms.com/docs/setup/installation#queue-setup

I don't know why the processes are hanging. I never saw that before, I'm using this plugin for 2 years without a single problem with over thousand of subscribers.

**Comment by ZuggieTM** [comment]
Alright. Are you able to share your queue setup without any sensitive information of course?
It is the correct version. 7.2.
I have `* * * * * php /path/to/artisan schedule:run >> /dev/null 2>&1` and it runs other schedules fine and in htop it says it's running /usr/bin/php7.2, so it should be good

After going into the backend and pressing save and close, it actually closed all the queue workers and just started up a single one. Still odd though.

**Comment by TimFoerster** [comment]
I had the problem that the `php` bin gave me errors with cli.
So I just run the worker directly without using the scheduler at all.

`config/queue.php`
```
[...]
    'default' => env('QUEUE_DRIVER', 'sync'),
[...]
```

`.env`
```
[...]
QUEUE_DRIVER=database
[...]
```
`crontab -e`
```
* * * * * cd /xxx/dev && php7.1-cli artisan queue:work --queue=newsletter --once
* * * * * cd /xxx/production && php7.1-cli artisan queue:work --queue=newsletter --once
```

**Comment by ZuggieTM** [comment]
Alright, I'll play around with it.

Thank you for the fast replies :)
